bin/pfioh

```diff
@@ -21,11 +21,16 @@ import  pudb
 # and related dependencies.
 sys.path.insert(1, os.path.join(os.path.dirname(__file__), '../pfioh'))
 try:
-    import  pfioh
+    import pfioh
+    import mount_dir
```

Comment:
Just move the import of mount_dir and swift_store out of the try catch entirely.